test/API/titerate.c

```diff
@@ -93,6 +90,13 @@ liter_cb(hid_t H5_ATTR_UNUSED group, const char *name, const H5L_info2_t H5_ATTR
     static int count  = 0;
     static int count2 = 0;
 
+    if (!(vol_cap_flags_g & H5VL_CAP_FLAG_ITERATE)) {
+        SKIPPED();
+        HDprintf("    API functions for iterate aren't "
+                 "supported with this connector\n");
+        return 0;
```

Comment:
This link callback function shouldn't end up getting called if the cap flag checking is setup correctly, but in case it does it's probably better to return 1 here instead of 0 to signal that the callback should exit early with a success value instead of continuing for each link.

Reply:
Okay, got it.